geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/partitions/AttributeScheme.scala

```diff
@@ -22,8 +22,9 @@ import org.locationtech.geomesa.index.index.attribute.AttributeIndexKey
   * @param attribute attribute name
   * @param index attribute index in the sft
   * @param binding type binding
+  * @param enums list of enums to partition
   */
-case class AttributeScheme(attribute: String, index: Int, binding: Class[_]) extends PartitionScheme {
+case class AttributeScheme(attribute: String, index: Int, binding: Class[_], enums: Seq[String]) extends PartitionScheme {
```

Comment:
Please update the docs https://github.com/locationtech/geomesa/blob/main/docs/user/filesystem/partition_schemes.rst and add unit tests https://github.com/locationtech/geomesa/blob/main/geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/test/scala/org/locationtech/geomesa/fs/storage/common/partitions/PartitionSchemeTest.scala